### PR TITLE
feat: add stage env to allowed environments for kpis

### PR DESCRIPTION
**Pull request by peterD4N**

### CODE REVIEW

**Review by sigurdfalk — CHANGES_REQUESTED** [review]
Need to update the `/dist` bundle by running `npm run all`

**Comment by peterD4N** [comment]
> Need to update the `/dist` bundle by running `npm run all`

Thank you, updated 😉

File: src/main.ts [hunk 1]
```diff
@@ -2,7 +2,7 @@ import * as core from '@actions/core'
 import * as github from '@actions/github'
 import fetch from 'node-fetch';
 
-const allowedEnvs = ['sand', 'dev', 'test', 'prod'];
+const allowedEnvs = ['sand', 'dev', 'test', 'stage', 'prod'];
```
Comment (sigurdfalk) [inline src/main.ts:5]: Not `staging`?

Comment (peterD4N) [reply src/main.ts:5]: I was thinking to use staging but then as we use e.g. `test` not `testing` I went for `stage`. But `staging` also works for me.

**Review by sivapalan — APPROVED** [review]

**Review by akorp — APPROVED** [review]

**Review by sigurdfalk — APPROVED** [review]

**Pull request merged**